From 97ee545b95ac7c723e342258e7244622067dbe3d Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Fri, 8 Oct 2021 13:27:45 +0000
Subject: [PATCH 0974/1000] MANTIS 0061647 extension: 0061647:
 Conveyor-Component: new mode for conveyor: maintenance, forceBag, off
 codes.idl V1.5.2

- added status codes BAGGAGE_MAINTENANCE_MODE, BAGGAGE_FORCEBAG_MODE, BAGGAGE_OFF_MODE


git-svn-id: svn://localhost/SelfServiceCommon/trunk@1676 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Cuss/codes.idl | 21 ++++++++++++++-------
 1 file changed, 14 insertions(+), 7 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Cuss/codes.idl b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
index 1621ca4c..7fa10098 100644
--- a/SelfServiceCommon/Massai/idl/Cuss/codes.idl
+++ b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
@@ -26,30 +26,30 @@ Changed:   Added baggage handling status codes (Nicolas Bruns, 29.06.2004)
 @date 26.11.2013
 @li V1.3.1
 @li Mantis 0029957: ConveyorComp: CUSS1.3 Conveyor (without ConveyorAEA)
-@li use full CUSS1.3 IDL + Materna extension statuscodes: BAGGAGE_INTRUSION, BAGGAGE_INTRUSION_FRONT, BAGGAGE_INTRUSION_BACK, 
+@li use full CUSS1.3 IDL + Materna extension statuscodes: BAGGAGE_INTRUSION, BAGGAGE_INTRUSION_FRONT, BAGGAGE_INTRUSION_BACK,
 
 @date 26.01.2016
 @li V1.3.2
 @li Mantis 0037350: integration of EZ-Access device for KIS kiosk
 @li added DS_TYPES_KEY, DS_TYPES_KEY_DOWN, DS_TYPES_KEY_UP
 @li the values are temporary, until defined in CUSS; see Mantis 0037186: CUSS-Spec: missing definition of DS_TYPES_KEY
-               
+
 @date 27.01.2016
 @li V1.3.3
 @li Mantis 0037186: CUSS-Spec: missing definition of DS_TYPES_KEY
 @li changed values for DS_TYPES_KEY, DS_TYPES_KEY_DOWN, DS_TYPES_KEY_UP
-               
+
 @date 17.02.2016
 @li V1.3.4
 @li Mantis 0040061: Extend Conveyor-Component for extended ConveyorDev.idl
 @li added BAGGAGE_INCORRECTLY_PLACED, BAGGAGE_TUB_REQUIRED, BAGGAGE_HANDLE_EXTENDED, BAGGAGE_TOO_WIDE
-               
+
 @date 18.02.2016
 @li V1.3.5
 @li Mantis 0037187: CUSS-Spec: DS_TYPES_SSML11= 25001
 @li added DS_TYPES_SSML10; DS_TYPES_SSML11
 - clarified: use DS_TYPES_SSML11= 25010 instead of DS_TYPES_SSML11= 25001
-               
+
 @date 19.02.2016
 @li V1.3.6
 @li Mantis 0040061: Extend Conveyor-Component for extended ConveyorDev.idl
@@ -67,7 +67,7 @@ Changed:   Added baggage handling status codes (Nicolas Bruns, 29.06.2004)
 - difference to original
   - added DS_TYPES_KEY, DS_TYPES_KEY_UP, DS_TYPES_KEY_DOWN
   - fixed DS_TYPES_SSML_11
-  - added materna extension BAGGAGE_INTRUSION, BAGGAGE_INTRUSION_FRONT, BAGGAGE_INTRUSION_BACK, BAGGAGE_TRANSPORT_BUSY, BAGGAGE_MISTRACKED, BAGGAGE_UNEXPECTED_CHANGE, BAGGAGE_ACCEPTED, BAGGAGE_DELIVERED, BAGGAGE_INTERFERENCE_USER, BAGGAGE_INTRUSION_SAFETY, BAGGAGE_INCORRECTLY_PLACED, BAGGAGE_TUB_REQUIRED, BAGGAGE_HANDLE_EXTENDED, BAGGAGE_TOO_WIDE, BAGGAGE_TOO_SMALL,               
+  - added materna extension BAGGAGE_INTRUSION, BAGGAGE_INTRUSION_FRONT, BAGGAGE_INTRUSION_BACK, BAGGAGE_TRANSPORT_BUSY, BAGGAGE_MISTRACKED, BAGGAGE_UNEXPECTED_CHANGE, BAGGAGE_ACCEPTED, BAGGAGE_DELIVERED, BAGGAGE_INTERFERENCE_USER, BAGGAGE_INTRUSION_SAFETY, BAGGAGE_INCORRECTLY_PLACED, BAGGAGE_TUB_REQUIRED, BAGGAGE_HANDLE_EXTENDED, BAGGAGE_TOO_WIDE, BAGGAGE_TOO_SMALL,
 @date 15.04.2016
 @li V1.4.1
 @li Mantis 0040920: Conveyor3Dev: additional error codes ManualMode; SwitchedOff
@@ -103,6 +103,10 @@ Changed:   Added baggage handling status codes (Nicolas Bruns, 29.06.2004)
 @li Mantis 0056794: MDCI - Support Media Type "ID_DOCUMENT"
 - added data status DS_TYPES_ID_DOCUMENT
 
+@date 08.10.2021
+@li V1.5.2
+@li Mantis 0061647: Conveyor-Component: new mode for conveyor: maintenance, forceBag, off
+- added status codes BAGGAGE_MAINTENANCE_MODE, BAGGAGE_FORCEBAG_MODE, BAGGAGE_OFF_MODE
 
 */
 
@@ -242,7 +246,7 @@ module datastatus
     const long DS_TYPES_CYPHER_CHALLENGE         = 40030; /**< random challenge for key generation */
     const long DS_TYPES_CYPHER_KEYS              = 40040; /**< encryption keys */
     const long DS_TYPES_CYPHER_TRACK             = 40050; /**< encrypted track data */
-	
+
 };
 
 /**
@@ -401,6 +405,9 @@ module statuscodes
     const long BAGGAGE_TOO_SMALL             = 174; /**< <b>Scope:</b> private + platform; materna ccm extension */
     const long BAGGAGE_MANUAL_MODE           = 175; /**< <b>Scope:</b> private + platform; materna ccm extension */
     const long BAGGAGE_SWITCHED_OFF          = 176; /**< <b>Scope:</b> private + platform; materna ccm extension */
+    const long BAGGAGE_MAINTENANCE_MODE      = 177; /**< <b>Scope:</b> private + platform; materna den extension */
+    const long BAGGAGE_FORCEBAG_MODE         = 178; /**< <b>Scope:</b> private + platform; materna den extension */
+    const long BAGGAGE_OFF_MODE              = 179; /**< <b>Scope:</b> private + platform; materna den extension */
 
 
     const long FORMAT_ERROR                = 201; /**< <b>Scope:</b> private + platform (platform only for output classes) */
-- 
2.41.0.windows.1

